src/biogeophys/HydrologyNoDrainageMod.F90

```diff
@@ -453,8 +458,16 @@ subroutine HydrologyNoDrainage(bounds, &
       end do
 
       ! Calculate column average snow depth
+      ! Use updated snow cover fraction to account for
+      ! surface water to ice conversion and frost deposition
+      ! that may have occurred since initial frac_sno calculation
       do c = bounds%begc,bounds%endc
-         snowdp(c) = snow_depth(c) * frac_sno_eff(c)
+         frac_sno_update = frac_sno_eff(c) &
+              + tanh(0.1 * (qflx_h2osfc_to_ice(c)+qflx_soliddew_to_top_layer(c))*dtime) &
```

Comment:
the 0.1 should be the accum_factor from SnowCoverFractionSwensonLawrence2012Mod, but currently it is a private parameter.  What's the best way to use that parameter here?